docs/api/_index.md

```diff
@@ -0,0 +1,749 @@
+---
+title: "HTTP API"
+linkTitle: "HTTP API"
+weight: 5
+slug: api
+menu:
+no_section_index_title: true
+---
+
+Cortex exposes an HTTP API for pushing and querying time series data, other then operating the cluster itself.
+
+For the sake of clarity, in this document we have grouped API endpoints by service, but keep in mind that they're exposed both when running Cortex in microservices and singly-binary mode:
+- **Microservices**: each service exposes its own endpoints
+- **Single-binary**: the Cortex process exposes all API endpoints for the services running internally
+
+## Endpoints
+
+| API | Service | Endpoint |
+| --- | ------- | -------- |
+| [Index page](#index-page) | _All services_ | `GET /` |
+| [Configuration](#configuration) | _All services_ | `GET /config` |
+| [Services status](#services-status) | _All services_ | `GET /services` |
+| [Readiness probe](#readiness-probe) | _All services_ | `GET /ready` |
+| [Remote write](#remote-write) | Distributor | `POST /api/v1/push` |
+| [Tenants stats](#tenants-stats) | Distributor | `GET /distributor/all_user_stats` |
+| [HA tracker status](#ha-tracker-status) | Distributor | `GET /distributor/ha_tracker` |
+| [Flush chunks / blocks](#flush-chunks--blocks) | Ingester | `GET|POST /ingester/flush` |
+| [Shutdown](#shutdown) | Ingester | `GET|POST /ingester/shutdown` |
+| [Ingesters ring status](#ingesters-ring-status) | Ingester | `GET /ingester/ring` |
+| [Instant query](#instant-query) | Querier, Query-frontend | `GET|POST <prometheus-http-prefix>/api/v1/query` |
```

Comment:
`GET|POST <prometheus-http-prefix>/api/v1/query` is broken in table.

Reply:
See https://github.com/cortexproject/cortex/pull/3050/files#r473089054